simplq/src/components/common/CreateJoinForm/CreateJoinForm.jsx

```diff
@@ -61,18 +56,14 @@ const CreateJoinForm = (props) => {
         />
       </div>
       <div className={styles['button-group']}>
-        {createInProgress ? (
-          <LoadingIndicator />
-        ) : (
-          <>
-            <div>
-              <StandardButton onClick={handleCreateClick}>Create Queue</StandardButton>
-            </div>
-            <div>
-              <StandardButton onClick={handleJoinClick}>Join Queue</StandardButton>
-            </div>
-          </>
-        )}
+        <Loading isLoading={actionStatus === 'pending'}>
```

Comment:
You can use `<LoadingStatus />` here.  Check [this](https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/pages/Join/JoinPage.jsx#L35) out.

Reply:
LoadingStatus doesn't work here in its current form. We need the buttons to be displayed at the start and only show the Loading component after click, but since the CreateQueue action isn't triggered on page load, the corresponding actionStatus resolves to undefined, so the Loading component is always displayed instead of the children. We'll need to handle this case in LoadingStatus to be able to use it better.

Reply:
Got it. It makes sense to spin only on 'pending' and `isLoading=true` [here](https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/common/Loading/Loading.jsx). 
```
if (actionStatus === 'rejected') { do current behavious, show error }
if (actionStatus === 'pending' || isLoading === true) { spin }
else render children
```

Reply:
Yes, I think this will not cause any problems. I'll make the changes